Classes/Backend/Preview/WallsIoBackendPreviewRenderer.php

```diff
@@ -209,32 +243,35 @@ protected function renderContentElementPreviewFromFluidTemplate(array $row, ?Gri
         if ($fluidTemplateFileAbsolutePath === '') {
             return null;
         }
+
         try {
             $view = GeneralUtility::makeInstance(StandaloneView::class);
             $view->setTemplatePathAndFilename($fluidTemplateFileAbsolutePath);
             $view->assignMultiple($row);
-            if ($table === 'tt_content' && !empty($row['pi_flexform'])) {
+            if (!empty($row['pi_flexform'])) {
```

Comment:
xml2array within FlexFormService expects FlexForm to be "string". So we can check against string here.
```suggestion
            if ($row['pi_flexform'] !== '') {
```